src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -5138,14 +5138,23 @@ namespace HeatBalanceManager {
         SetupOutputVariable(
             "Zone Outdoor Air Wind Direction", OutputProcessor::Unit::deg, Zone(ZoneLoop).WindDir, "Zone", "Average", Zone(ZoneLoop).Name);
 
-        if (FlagHybridModel) {
-            SetupOutputVariable("Zone Infiltration Hybrid Model Air Change Rate",
-                                OutputProcessor::Unit::ach,
-                                Zone(ZoneLoop).InfilOAAirChangeRateHM,
-                                "Zone",
-                                "Average",
-                                Zone(ZoneLoop).Name);
-        }
+
+		// Modified by Han Li 07/02/2018
+        SetupOutputVariable("Zone Infiltration Hybrid Model Air Change Rate", OutputProcessor::Unit::ach, Zone(ZoneLoop).InfilOAAirChangeRateHM,
+                            "Zone", "Average", Zone(ZoneLoop).Name);
+
+        // Added by Han li 07/20/2018
+        SetupOutputVariable("Zone Hybrid Model People Count", OutputProcessor::Unit::None, Zone(ZoneLoop).NumOccHM,
+                            "Zone", "Average", Zone(ZoneLoop).Name);
+
+		SetupOutputVariable("Zone Infiltration Hybrid Model Mass Flow Rate", OutputProcessor::Unit::kg_s, Zone(ZoneLoop).MCPIHM,
+			"Zone", "Average", Zone(ZoneLoop).Name);
+
+        // Test--
+		SetupOutputVariable("Zone ZoneMeasuredHumidityRatio", OutputProcessor::Unit::kg_kg, Zone(ZoneLoop).ZoneMeasuredHumidityRatio, "Zone", "Average", Zone(ZoneLoop).Name);
+        SetupOutputVariable("Zone ZoneMeasuredTemperature", OutputProcessor::Unit::C, Zone(ZoneLoop).ZoneMeasuredTemperature, "Zone", "Average", Zone(ZoneLoop).Name);
+        SetupOutputVariable("Zone ZoneMeasuredCO2Concentration", OutputProcessor::Unit::ppm, Zone(ZoneLoop).ZoneMeasuredCO2Concentration, "Zone", "Average", Zone(ZoneLoop).Name);
+
```

Comment:
To use the flag, I put the output variables here:
https://github.com/NREL/EnergyPlus/pull/7130/commits/858ad1d09c9dcf24fe0ced8fbe489db182869948